test/relativistic/spacetime_interval_is_lorentz_invariant_test.py

```diff
@@ -0,0 +1,31 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from sympy import I
+from symplyphysics import (
+    assert_equal,
+    errors,
+    units,
+    Quantity,
+)
+from symplyphysics.laws.relativistic import spacetime_interval_is_lorentz_invariant as invariant_law
+
+Args = namedtuple("Args", "s")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    s = Quantity(5 * I * units.meter)
```

Comment:
It's a bit unusual to see complex number here. Add small description, what does it mean.